### PR TITLE
Bump Gradle to 8.0.2

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

**Comment by snazy** [comment]
Hope this PR is rather "boring" ;)

File: devtools/gradle/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=948d1e4ccc2f6ae36cbfa087d827aaca51403acae5411e664a6000bb47946f22
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-all.zip
+ddistributionSha256Sum=47a5bfed9ef814f90f8debcbbb315e8e7c654109acd224595ea39fca95c5d4da
```
Comment (aloubyansky) [inline devtools/gradle/gradle/wrapper/gradle-wrapper.properties:3]: Double `d`?

**Comment by snazy** [comment]
Weird test failures. Rebased + ran some of these tests locally (passed fine).

**Review by aloubyansky — APPROVED** [review]

**Comment by aloubyansky** [comment]
There still seem to be issues on Windows.

**Comment by gsmet** [comment]
The Gradle tests have been unstable on Windows for a while. We have a memory issue I think. @snazy if at some point you could have a look that would be awesome.

**Pull request merged**